opslevel/resource_opslevel_service.go

```diff
@@ -188,13 +188,13 @@ func reconcileTags(d *schema.ResourceData, service *opslevel.Service, client *op
 func resourceServiceCreate(d *schema.ResourceData, client *opslevel.Client) error {
 	// owner_alias is deprecated, allow user to use one or the other but not both.
 	var ownerField *opslevel.IdentifierInput
-	owner, ownerUsed := d.GetOk("owner")
-	ownerAlias, ownerAliasUsed := d.GetOk("owner_alias")
-	if ownerUsed && ownerAliasUsed {
+	owner := d.Get("owner")
```

Comment:
I replaced `GetOk()` with `Get()`. Reason being

>GetOk returns the data for the given key and whether or not the key **has been set to a non-zero value at some point.**

>The first result will not necessarilly be nil if the value doesn't exist. **The second result should be checked to determine this information.**

GetOk() isn't useful here, we still need to check the value. We need to see if the value is currently set, not if it was ever set before. 

---

Get() is the better function to use, because

>If the key does exist in the schema but doesn't exist in the configuration, then the default value for that type will be returned. For strings, this is "", for numbers it is 0, etc.

---

Source: https://pkg.go.dev/github.com/hashicorp/terraform/helper/schema#ResourceData.Get

Reply:
👍  - this might be a logic problem across the codebase then - lets make a ticket to audit this.

Reply:
Added new ticket to backlog: https://github.com/OpsLevel/team-platform/issues/152